### ISSUE TITLE
error when calling drino

### ISSUE THREAD

**Issue by tchereau** [remscodes/renault-api-client#74]
```bash
/node_modules/@remscodes/renault-api-client/src/gigya/gigya-client.ts:18
    this.httpClient = drino.create({
                            ^
TypeError: t.create is not a function
```
How to reproduce:
``npm init -y``
_setup normal typescript env_
``npm install @remscodes/renault-api-client``
copy past the example then run
``npx ts-node src/index.ts``

**Comment by remscodes** [comment]
Hi
Can you provide your `tsconfig.json` and `package.json` please.
The problem seems to be related to CommonJS/ESM files imports.
I'm continuing to dig into the potential errors caused by its dependencies.

**Comment by tchereau** [comment]
Hi, sure !

here is ``tsconfig.json``
```json
{
  "compilerOptions": {
    "module": "commonjs",
    "esModuleInterop": true,
    "target": "es6",
    "moduleResolution": "node",
    "sourceMap": true,
    "outDir": "dist"
  }
}
```

and here is my ``package.json``(I removed renault-api-client since I can't use it and installed renault-api to just have the url)
```json
{
  "name": "ohmyzoe_service",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "start": "node dist/index.js",
    "dev": "nodemon src/index.ts",
    "build": "tsc -p .",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "@types/express": "^4.17.21",
    "typescript": "^5.3.3"
  },
  "dependencies": {
    "@remscodes/renault-api": "^1.2.2",
    "dotenv": "^16.3.1",
    "express": "^4.18.2",
    "nodemon": "^3.0.2",
    "reflect-metadata": "^0.2.1",
    "ts-node": "^10.9.2",
    "typedi": "^0.10.0"
  }
}
```

**Comment by remscodes** [comment]
@tchereau I've made a release [v1.1.1](https://github.com/remscodes/renault-api-client/releases/tag/v1.1.1) that fixes the problem (PR: #76).

The issue was indeed related to `drino` where the default export available in ESM was not available in CommonJS. Now it's fixed. Thank you for highlighting it.

I've tested the new release with your `tsconfig.json` and `package.json` and it works fine.
Can you confirm that it also works on your side too ?

**Comment by tchereau** [comment]
Hi sorry for the delay, I was kinda busy and haven't time for this project
anyway, yes it is working now, thanks you :)
